ark/segmentation/marker_quantification_test.py

```diff
@@ -124,39 +136,50 @@ def test_compute_marker_counts():
 
 
 def test_generate_expression_matrix():
-    cell_mask, channel_data = _create_test_extraction_data()
+    fovs = [f'Point{i}' for i in range(2)]
+    chans = [f'chan{i}' for i in range(5)]
+
+    cell_mask, channel_data = test_utils.create_test_extraction_data()
 
     # generate data for two fovs offset
     cell_masks = np.zeros((2, 40, 40, 1), dtype="int16")
     cell_masks[0, :, :, 0] = cell_mask
     cell_masks[1, 5:, 5:, 0] = cell_mask[:-5, :-5]
 
-    channel_datas = np.zeros((2, 40, 40, 5), dtype="int16")
-    channel_datas[0, :, :, :] = channel_data
-    channel_datas[1, 5:, 5:, :] = channel_data[:-5, :-5]
-
-    segmentation_masks = xr.DataArray(cell_masks,
-                                      coords=[["Point1", "Point2"], range(40), range(40),
-                                              ["whole_cell"]],
-                                      dims=["fovs", "rows", "cols", "compartments"])
-
-    channel_data = xr.DataArray(channel_datas,
-                                coords=[["Point1", "Point2"], range(40), range(40),
-                                        ["chan0", "chan1", "chan2", "chan3", "chan4"]],
-                                dims=["fovs", "rows", "cols", "channels"])
-
-    normalized, arcsinh = marker_quantification.generate_expression_matrix(segmentation_masks,
-                                                                           channel_data)
+    tif_data = np.zeros((2, 40, 40, 5), dtype="int16")
+    tif_data[0, :, :, :] = channel_data
+    tif_data[1, 5:, 5:, :] = channel_data[:-5, :-5]
+
+    segmentation_masks = test_utils.make_labels_xarray(
+        label_data=cell_masks,
+        fov_ids=fovs,
```

Comment:
We should have fov_ids default to be what you added in line 139 above so that it only needs to be passed in if it's being changed